### PR TITLE
Handle redirects

### PR DESCRIPTION
**Pull request by sanjithkumar017**

### CODE REVIEW

File: src/utils/unbxdCallBack.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ function unbxdCallBack(unbxdSearchObj, eventName, data) {
         });
         const { onRouteChange } = this.props;
         if (typeof onRouteChange === 'function') {
-            onRouteChange(unbxdSearchObj.getStateString());
+            onRouteChange(unbxdSearchObj, unbxdSearchObj.getStateString());
```
Comment (vikramgupta1988) [inline src/utils/unbxdCallBack.js:12]: Why pass both, we can get second from first param, Right?

File: demo/src/pages/Search.js [hunk 2]
```diff
@@ -61,12 +66,38 @@ const Search = () => {
...
-            routeHistory.push(`${routeLocation.pathname}#${hash}`);
+        setEnableFilters(true);
+        const { state = {} } = searchObj;
+        const { responseObj = {} } = state;
+        const { redirect = {} } = responseObj;
+        const { type = '', value } = redirect;
+        const urlParams = searchObj.getQueryParams();
+        if (
+            routeLocation.pathname === '/' &&
+            Object.keys(urlParams).length === 0 &&
```
Comment (vikramgupta1988) [inline demo/src/pages/Search.js:79]: Can you add few code comments for if-else statements.

**Review by vikramgupta1988 — CHANGES_REQUESTED** [review]

File: src/utils/unbxdCallBack.js (hunk 1 above)
Comment (sanjithkumar017) [reply src/utils/unbxdCallBack.js:12]: Yes we can, but I felt it would be handy if we passed it as separate param.

Comment (vikramgupta1988) [reply src/utils/unbxdCallBack.js:12]: ok

**Pull request merged**